src/sass/formio.form.scss

```diff
@@ -1235,6 +1235,32 @@ div[data-oembed-url] {
   img {
     max-width: 100%;
   }
+
+  li[data-list="bullet"] {
```

Comment:
Isn't this CSS included in Quill?  This seems REALLY strange we have to include these styles in our renderer. The renderer should only be concerned with styles related to the rendering of forms.

Reply:
In read-only mode we do not load quill or any other editor, moreover, we render the value using Well component, that's why styling differs